modules/apps/layout/layout-type-controller/layout-type-controller-test/src/testIntegration/java/com/liferay/layout/type/controller/asset/display/internal/portlet/test/AssetDisplayPageFriendlyURLProviderImplTest.java

```diff
@@ -140,6 +165,23 @@ private void _assertGetFriendlyURL(String urlSeparator)
 				LocaleUtil.getSiteDefault(), _themeDisplay));
 	}
 
+	private void _assertGetFriendlyURL(
+			String urlSeparator, JournalArticle journalArticle)
+		throws PortalException {
+
+		Assert.assertEquals(
+			StringBundler.concat(
+				_portal.getGroupFriendlyURL(
+					_group.getPublicLayoutSet(), _themeDisplay, false, false),
+				urlSeparator,
+				journalArticle.getUrlTitle(LocaleUtil.getSiteDefault())),
+			_assetDisplayPageFriendlyURLProvider.getFriendlyURL(
+				new InfoItemReference(
+					JournalArticle.class.getName(),
+					journalArticle.getResourcePrimKey()),
+				LocaleUtil.getSiteDefault(), _themeDisplay));
+	}
+
```

Comment:
Can you please just use the new `_assertGetFriendlyURL` and remove the previous `_assertGetFriendlyURL` ?

Reply:
Hi @marcogalluzzi, I tried to do that originally, but for some reason, the tests keep failing if I use the new assertMethod. I even tried to use the old one as a wrapper to the new one, and it still failed. That is why I left it in.